apps/voting/contracts/Voting.sol

```diff
@@ -125,50 +132,65 @@ contract Voting is IForwarder, AragonApp {
     */
     function unsafelyChangeVoteTime(uint64 _voteTime)
         external
-        authP(UNSAFELY_MODIFY_VOTE_TIME_ROLE, arr(uint256(_voteTime), uint256(voteTime)))
+        auth(UNSAFELY_MODIFY_VOTE_TIME_ROLE)
     {
         voteTime = _voteTime;
 
         emit ChangeVoteTime(_voteTime);
     }
 
+    /**
+    * @notice Change objection time to `_objectionTime` sec. The change affects all existing unexecuted votes, so be really careful with it
```

Comment:
@notice Change the objection phase duration to `_objectionTime` sec. The change affects all existing unexecuted votes, so be really careful with it